bindings/swift/native/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[workspace]
+
+[package]
+name = "iota-wallet-swift-binding"
+version = "0.1.0"
+authors = ["Thoralf Müller <thoralf.mueller@iota.org", "Pasquale Ambrosini <pasquale.ambrosini@gmail.com"]
+edition = "2018"
+
+[lib]
+name = "wallet"
+crate-type = ["cdylib", "staticlib"]
+
+[dependencies]
+tokio = { version = "1.11", features = ["full"] }
```

Comment:
Not very good to go for a full feature set for a library: https://docs.rs/tokio/latest/tokio/#authoring-libraries